Cargo.toml

```diff
@@ -21,21 +22,26 @@ hmac = "0.11.0"
 http = "0.2"
 http-types = "2.5"
 humantime-serde = "1.0"
-isahc = "1.3"
+isahc = "1.6"
 lazy_static = "1.4.0"
 log = "0.4"
 mime_guess = "2.0"
-nu = { version = "0.42.0", features = ["extra"] }
-nu-protocol = { version = "0.42.0" }
-nu-data = { version = "0.42.0" }
-nu-errors = { version = "0.42.0" }
-nu-stream = { version = "0.42.0" }
-nu-source = { version = "0.42.0" }
-nu-cli = { version = "0.42.0" }
-nu-engine = { version = "0.42.0" }
-nu-plugin = { version = "0.42.0" }
-nu_plugin_from_bson = { version = "0.42.0" }
-nu-value-ext = { version = "0.42.0" }
+nu = { path = "../nushell", features = ["extra"] }
```

Comment:
These will obviously need to change, once my changes are merged upstream.

Reply:
I think that's done now, right?  Need to update this PR?

Reply:
The PR has been updated already, it was using local paths but is now using a git SHA. Nushell 0.60 is out now so once this PR is in we can make a new patchset to bump to 0.60 and use a proper tagged version again.